include/picongpu/plugins/output/ConstSpeciesAttributes.hpp

```diff
@@ -0,0 +1,88 @@
+/* Copyright 2020-2021 Richard Pausch, Rene Widera
```

Comment:
Since this file is substantiallz a copy of previouslz existing HDF5 plugin code, I think better to include authors of that old file here